packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -43,6 +44,15 @@ function warnOnAuth(context, map) {
   }
 }
 
+class TransformerConfVersionNotSupportedError extends Error {
+  constructor() {
+    super(
+      "The transformer conf version entered is higher than the current supported version."
```

Comment:
I think this message should be changed, Who is entering the transformer conf version? Probably the message should be
```
This project was pushed with newer version of Amplify CLI. Using {curreentVersion} would cause some features in the API to be removed 
```